### ISSUE TITLE
Multiline Code Block

### ISSUE THREAD

**Issue by adrianjost** [schul-cloud/edtrio#100]
The code plugin seems to not support multiline code :/
It looks pretty strange if the code is seperated that much.

![image](https://user-images.githubusercontent.com/22987140/47917219-0e90a580-dea9-11e8-82cf-9e9be3faf942.png)

**Comment by FWirtz** [comment]
yeah should be quite simple just need to suppress line breaks in there, maybe I'll find some time for it this week

**Comment by FWirtz** [comment]
code block is still wip also cuz it doesn't have syntax highlighting either
